src/Staged/GHC/Generics/GHCish.hs

```diff
@@ -0,0 +1,149 @@
+{-# language AllowAmbiguousTypes #-}
```

Comment:
:(

Reply:
I can try using proxies a bit differently if you prefer.

Reply:
It's fine. I just don't like when I don't know where the ambiguities are.

Here it doesn't matter as public interface is not ambiguous, AFAICT.